addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1652,6 +1652,63 @@ var SnippetsMenu = Widget.extend({
         ev.stopPropagation();
         this._activateSnippet(ev.data.$snippet.parent());
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onInstallBtnClick: function (ev) {
+        var self = this;
+        var $snippet = $(ev.currentTarget).closest('[data-module-id]');
+        var moduleID = $snippet.data('moduleId');
+        var name = $snippet.attr('name');
+        new Dialog(this, {
+            title: _t(`Install "${name}"`),
+            size: 'medium',
+            $content: $('<div/>', {text: _t(`Do you want to install the "${name}" App?`)}).append(
```

Comment:
Sorry @rdeodoo but in fact translating template literals do not work, we have to use `sprintf`
```javascript
`Hello ${world}`
```
=>
```javascript
_.str.sprintf(_t("Hello %s"), world)
```

Reply:
@rdeodoo Don't forget about this :p

Reply:
@rdeodoo Merging this today? :p

Reply:
Yes I wanted to finish the countdown before switching branch but this is now done